spec/spec_helper.rb

```diff
@@ -11,35 +12,29 @@
 require 'napa'
 require 'napa/rspec_extensions/response_helpers'
 
+module NapaSpecClassHelpers
+  # Apparently Thor's last name is Odinson... who knew?!
+  # http://marvel.com/universe/Thor_(Thor_Odinson)
+  # http://www.infinitelooper.com/?v=4D7cPH7DHgA#/170;230
+  def tell_me_mr_odinson_what_good_is_a_phone_call_if_youre_unable_to_speak?
```

Comment:
From context, I assume you want me to change the method name, but that comment seems passive aggressive, so I'm not going to. You're just as capable of changing it as I am, you could have merged it, changed it, pushed it, a month ago, with three minutes worth of effort. I'm not going to sanitize the humour for you.